day2/input.txt

```diff
@@ -0,0 +1,1000 @@
+4x23x21
```

Comment:
This file is kind of huge but not _that_ huge ... should I:
1. leave it as is?
2. compress it?
    1. should the rust code decompress it?
    2. should it just be mentioned in the readme to decompress it?
3. remove it?

I lean towards 1 for now but am cool with 2. 3 would be a bit of a bummer because we couldn't easily benchmark/test 🤷

Reply:
I'm not seeing syntax highlighting for the Rust files ... is it because of this big file?

Reply:
Actually after reading [this blog post](https://blog.logrocket.com/rust-compression-libraries/) I'm now very interested in trying out [the `brotli` crate](https://github.com/dropbox/rust-brotli) so now I lean towards 2 with the rust code decompressing it (and, presumably, a separate binary crate for compressing the input originally).

Reply:
Going to not compress things for now 👍